analysis_rsp.tex

```diff
@@ -0,0 +1,50 @@
+
+\subsubsection{Portal}
```

Comment:
Add schema browser, HiPS browsing, i.e., data discoverability as Portal analysis use cases. 

Add cutouts and query history and user table uploads.